articles/migrations/guides/calling-api-with-idtokens.md

```diff
@@ -0,0 +1,298 @@
+  "headers": [{
+    "name": "Authorization",
+    "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+  }]
+}
+```
+
+### The Token endpoint
+
+In this section we will use an example to showcase the differences in how you get a token with the [Token endpoint](/api/authentication#get-token). Keep in mind though that no matter which endpoint you want to migrate, the changes are the same, the only thing that differs is the [scopes](#changes-in-scopes) you will specify in the request.
+
+In the example below, we want to use the [GET User by ID endpoint](/api/management/v2#!/Users/get_users_by_id) to retrieve the full profile information of the logged-in user. To do so, first we will authenticate our user (using the [Password Exchange grant](/api/authentication?http#resource-owner-password)) and retrieve the token(s).
+
+On the `Legacy (ID Token)` script you can see an implementation of the old approach that gets an ID Token (and then uses it to call the endpoint). On the `Current (Access Token)` script you can see the new approach that gets an Access Token as well.
+
+<div class="code-picker">
+  <div class="languages-bar">
+    <ul>
+      <li class="active"><a href="#token-id-token" data-toggle="tab">Legacy (ID Token)</a></li>
+      <li><a href="#token-access-token" data-toggle="tab">Current (Access Token)</a></li>
+    </ul>
+  </div>
+  <div class="tab-content">
+    <div id="token-id-token" class="tab-pane active">
+      <pre class="text hljs">
+        <code>
+POST https://${account.namespace}/oauth/token
+Content-Type: application/json
+{
+  "grant_type": "password",
+  "username": "USERNAME",
+  "password": "PASSWORD",
+  "scope": "openid",
+  "client_id": "${account.clientId}",
+  "client_secret": "YOUR_CLIENT_SECRET",
+}
+        </code>
+      </pre>
+    </div>
+    <div id="token-access-token" class="tab-pane">
+      <pre class="text hljs">
+        <code>
+POST https://${account.namespace}/oauth/token
+Content-Type: application/json
+{
+  "grant_type": "password",
+  "username": "USERNAME",
+  "password": "PASSWORD",
+  "audience": "https://${account.namespace}/api/v2/",
+  "scope": "read:current_user",
+  "client_id": "${account.clientId}",
+  "client_secret": "YOUR_CLIENT_SECRET",
+}
+        </code>
+      </pre>
+    </div>
+  </div>
+</div>
+
+In order to get an Access Token that can access the Management API:
+- We set the `audience` to `https://${account.namespace}/api/v2/`
+- We asked for the scope `read:current_user`
+
+Once you have the Access Token you can use it to call the endpoint. This part remains the same, nothing else changes in the request except for the value you use as `Bearer` token. The response remains also the same.
+
+```har
+{
+  "method": "GET",
+  "url": "https://${account.namespace}/api/v2/users/USER_ID",
+  "headers": [{
+    "name": "Authorization",
+    "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+  }]
+}
+```
+
+### Lock or auth0.js (embedded)
```

Comment:
Is the example listed here valid also for Lock? If so let's put a p making that explicit. If not let's add one example for Lock.

Reply:
Goal is to have link that TSE (or anyone) can share with a customer that addresses the customer scenario.

Reply:
This is valid also for Lock in a sense, but it is done through Auth0.js. It's one of the cases where we recommend using them together:

> For simple uses, Lock is all that is necessary. However, while using Lock, if more customization is required in an application than Lock allows, functionality from the Auth0.js SDK can be used alongside Lock to meet those needs. An example might be using Lock to handle signups and logins, while using auth0.js to manage users (read and update user metadata, link user accounts together, and similar tasks).
https://auth0.com/docs/libraries/lock/v11/auth0js

I'll make it more clear in the text.